src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,45 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
+public class CreateDriverController extends HttpServlet {
+    private static Injector injector = Injector.getInstance("mate");
+    private static DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String driverName = req.getParameter("driver_name");
+        String licenseNumber = req.getParameter("licenseNumber");
+        List<Driver> drivers = driverService.getAll();
+        Driver driver = new Driver(driverName,licenseNumber);
+        for (Driver tempDriver : drivers) {
+            if (tempDriver.getName().equals(driver.getName())
+                    && tempDriver.getLicenseNumber()
+                    .equals(driver.getLicenseNumber())) {
+                throw new RuntimeException("Driver " + driver.getName()
+                        + " with licenseNumber " + driver.getLicenseNumber()
+                        + " is already exist");
+            }
+        }
```

Comment:
Same here. First of all we definetelly could have 2 drivers with the same name. Second - we could check the uniqueness of the license number, but we should create separate methods in dao and service layers to find driver by license number(or check if there is such license number in the db). And I think we should do this check on service layer, not in controller, as this is a part of business logic. (Same in all other places in the code)